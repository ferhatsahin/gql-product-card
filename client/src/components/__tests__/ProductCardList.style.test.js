import React from "react";
import { render, cleanup } from "../../test.utils";

import {
  NotFoundText,
  PageTitle,
  CardContainer
} from "../ProductCardList.style";

afterEach(cleanup);

test("NotFoundText renders", () => {
  const { container } = render(<NotFoundText />);
  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild).toHaveStyleRule("padding-left", "10px");
});

test("PageTitle renders", () => {
  const { container } = render(<PageTitle />);
  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild).toHaveStyleRule("padding-left", "10px");
});

test("CardContainer renders", () => {
  const { container } = render(<CardContainer />);
  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild).toHaveStyleRule("flex-wrap", "wrap");
});
