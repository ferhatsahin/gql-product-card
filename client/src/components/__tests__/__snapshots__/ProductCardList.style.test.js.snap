// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer renders 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

<div
  class="c0"
/>
`;

exports[`NotFoundText renders 1`] = `
.c0 {
  padding-left: 10px;
}

<p
  class="c0"
/>
`;

exports[`PageTitle renders 1`] = `
.c0 {
  font-size: 25px;
  padding-left: 10px;
  -webkit-align-self: flex-start;
  -ms-flex-item-align: start;
  align-self: flex-start;
}

<p
  class="c0"
/>
`;
